kornia/utils/image.py

```diff
@@ -176,21 +189,32 @@ def _wrapper(input: torch.Tensor, *args, **kwargs):
 
         input_shape = input.shape
         input = _to_bchw(input)  # view input as (B, C, H, W)
-        output: torch.Tensor = f(input, *args, **kwargs)
-        if len(input_shape) == 3:
-            output = output[0]
-
-        if len(input_shape) == 2:
-            output = output[0, 0]
-
-        if len(input_shape) > 4:
-            output = output.view(*(input_shape[:-3] + output.shape[-3:]))
+        output: Union[torch.Tensor, Tuple] = f(input, *args, **kwargs)
```

Comment:
I
```suggestion
        output = f(input, *args, **kwargs)
```
adding `Union` typing  might break jit support